agiverse/agent/memory/manager.py

```diff
@@ -0,0 +1,41 @@
+from typing import List, Optional, Dict
+import numpy as np
+from .base import Memory, MemoryStream
+from .storage import LocalStorage
+from datetime import datetime
+
+class MemoryManager:
+    def __init__(self, importance_calculator, embedding_generator):
+        self.importance_calculator = importance_calculator
+        self.embedding_generator = embedding_generator
+        self.storage = LocalStorage(persist_directory="data/memories")
```

Comment:
Don't hard code `data/`, use agent.data_dir or pass it to here so it's user configurable